public/app/plugins/datasource/cloudwatch/components/LogsCheatSheet.tsx

```diff
@@ -28,19 +29,19 @@ const CLIQ_EXAMPLES: QueryExample[] = [
       },
       {
         title: 'Determine the amount of overprovisioned memory',
-        expr: stripIndent`
-        filter @type = "REPORT" |
-        stats max(@memorySize / 1024 / 1024) as provisonedMemoryMB,
-              min(@maxMemoryUsed / 1024 / 1024) as smallestMemoryRequestMB,
-              avg(@maxMemoryUsed / 1024 / 1024) as avgMemoryUsedMB,
-              max(@maxMemoryUsed / 1024 / 1024) as maxMemoryUsedMB,
-              provisonedMemoryMB - maxMemoryUsedMB as overProvisionedMB`,
+        expr: stripIndent`filter @type = "REPORT"
+        | stats max(@memorySize / 1000 / 1000) as provisionedMemoryMB,
+          min(@maxMemoryUsed / 1000 / 1000) as smallestMemoryRequestMB,
+          avg(@maxMemoryUsed / 1000 / 1000) as avgMemoryUsedMB,
+          max(@maxMemoryUsed / 1000 / 1000) as maxMemoryUsedMB,
+          provisionedMemoryMB - maxMemoryUsedMB as overProvisionedMB
+        `,
```

Comment:
why change from `1024` to `1000`? is this what the cloudwatch samples are using?

Reply:
Yeah that's just what I see here, will defer to AWS's docs on this and assume they are right?
<img width="1213" alt="Screenshot 2023-04-25 at 4 00 52 PM" src="https://user-images.githubusercontent.com/6620164/234389662-f822c069-782d-456f-85d1-b8c71c632113.png">